Add missing meta descriptions (#488)
mongodb/docs-golang: source/issues-and-help.txt

.. meta::
   :keywords: suggestion, github

>    :description: Find support for the Go driver through community forums, report bugs or feature requests via JIRA, and contribute with pull requests following guidelines.

Reviewer: ```suggestion
   :description: Find support for the MongoDB Go Driver through community forums, report bugs or feature requests via JIRA, and create pull requests to contribute.
```
